### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by hamzakashif5289** [internetisgone/angel-downloader#1]
Not Working and giving 302

**Comment by internetisgone** [comment]
yeah seems that xiaohongshu has switched up their anti bot measures. if not logged in, a `xsec_token` query string will be required to access notes. i haven't tested but https://github.com/JoeanAmier/XHS-Downloader should still work, it allows u to set cookies in a config file

**Comment by hamzakashif5289** [comment]
can you please guide more how to setup please help me i really need it
\
